src/converge.js

```diff
@@ -1,22 +1,23 @@
+var _curry2 = require('./internal/_curry2');
 var _map = require('./internal/_map');
-var _slice = require('./internal/_slice');
 var curryN = require('./curryN');
 var pluck = require('./pluck');
 
 
 /**
- * Accepts at least three functions and returns a new function. When invoked, this new
- * function will invoke the first function, `after`, passing as its arguments the
- * results of invoking the subsequent functions with whatever arguments are passed to
- * the new function.
+ * Accepts a converging function and a list of branching functions and returns a new function.
+ * When invoked, this new function will invoke the first function, `after`, passing as its
+ * arguments the results of invoking the subsequent functions with whatever arguments are
+ * passed to the new function.
  *
  * @func
  * @memberOf R
  * @category Function
- * @sig (x1 -> x2 -> ... -> z) -> ((a -> b -> ... -> x1), (a -> b -> ... -> x2), ...) -> (a -> b -> ... -> z)
+ * @sig (x1 -> x2 -> ... -> z) -> ((a -> b -> ... -> x1), (a -> b -> ... -> x2), ...) ->
+ *      (a -> b -> ... -> z)
```

Comment:
Though it's nice to wrap the description, I don't think our documentation pipeline can handle wrapped type signatures. Let's revert this.

Also, the type signature should now include `[` and `]` to indicate that the branching functions are taken as a list, rather than as positional arguments.